services/parser-database/parser.py

```diff
@@ -0,0 +1,225 @@
+import logging
+import hashlib
+
+# import datetime
+# import requests
+import json
+
+import slate  # pylint: disable=import-error
+
+import retriever  # pylint: disable=import-error
+# import database  # pylint: disable=import-error
+import keywordmock  # pylint: disable=import-error
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Document:
+    """Class for Regulations Documents"""
+    def __init__(self, jurisdiction, shahash, url, last_updated):
+        self.id = None
+        self.jurisdiction = jurisdiction
+        self.shahash = shahash
+        self.url = url
+        self.last_updated = last_updated
+
+    def __repr__(self):
+        return f"Document(\
+                id={self.id}, \
+                jurisdiction={self.jurisdiction}, \
+                last_updated={self.last_updated}, \
+                url={self.url}, \
+                shahash={self.shahash}\
+            )"
+
+
+class Article:
+    """Class for storing articles.
+    """
+    def __init__(self, number, text):
+        self.number = number
+        self.text = text
+
+
+articles_in_memory = {}
+keywords_in_memory = {}
+
+
+def get_article_by_number(art_num):
+    if art_num in articles_in_memory:
+        return articles_in_memory[art_num]
+    return None
+
+
+def get_articles_that_match_keywords(keywords_list):
```

Comment:
Missing a unit test for this

Reply:
I do not know how to test this with mocking since I cannot guarantee I will always get the same keywords when mocking Javier's service.